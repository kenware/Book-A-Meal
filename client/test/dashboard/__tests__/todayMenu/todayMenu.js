import React from 'react';
import { shallow } from 'enzyme';
import TodayMenu from '../../../../components/dashboard/todayMenu';
import { props } from '../../dashboardMockData';

describe('Test Today menu component of menu', () => {
  it('renders correctly', () => {
    const tree = shallow(<TodayMenu {...props} />);
    expect(tree).toMatchSnapshot();
  });
  // it('should have three modal-order-content class', () => {
  //   const tree = shallow(<TodayMenu {...props} />);
  //   tree.find('.order1').simulate('click');
  //   expect(tree).toMatchSnapshot();
  // });
});
