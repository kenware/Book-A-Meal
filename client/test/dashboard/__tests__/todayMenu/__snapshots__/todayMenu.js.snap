// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Today menu component of menu renders correctly 1`] = `
<div>
  <div
    key="1"
  >
    <button
      className="menuAccordion"
      onClick={[Function]}
    >
      <span
        style={
          Object {
            "float": "right",
            "marginTop": "1.5rem",
          }
        }
      >
        <em
          className="fa fa-plus p-color"
        />
      </span>
      <img
        alt="profile"
        className="user-img rounded-circle"
        src="image/profile.png"
      />
      <br />
    </button>
    <div
      className="showA meal-menu-row spreadundefined"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-spinner fa-spin fa-4x fa-fw"
        />
      </div>
      <div
        className="meal-pagination"
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <a
          activePage={1}
          firstPageText="«"
          getPageUrl={[Function]}
          hideFirstLastPages={false}
          innerClass="pagination"
          itemsCountPerPage={4}
          lastPageText="»"
          nextPageText="⟩"
          onChange={[MockFunction]}
          pageRangeDisplayed={6}
          prevPageText="⟨"
          totalItemsCount={2}
        />
      </div>
    </div>
  </div>
</div>
`;
